src/Backups/BackupSettings.h

```diff
@@ -90,6 +90,9 @@ struct BackupSettings
     /// Is it allowed to use blob paths to calculate checksums of backup entries?
     bool allow_checksums_from_remote_paths = true;
 
+    /// A length of backup key prefix.
+    size_t key_prefix_length = 0;
```

Comment:
It seems more logical to have two separate settings - the first one to enable the approach of using checksums as data file names, and the second one to specify the prefix length. And if the prefix length is zero it seems more obvious just not to separate any prefix from checksum converted to strings.